### ISSUE TITLE
L210.resources_USA throwing message re precursors

### ISSUE THREAD

**Issue by bpbond** [JGCRI/gcamdata#847]
```
[1] "module_gcam.usa_L210.resources_USA"
[1] "- make 0.38"
Inputs L1321.out_Mt_state_cement_Yh don't appear as precursors for any outputs - chunk module_gcam.usa_L210.resources_USA
```

**Comment by russellhz** [comment]
I can fix this and just put it in #843 if that works for you @bpbond 

It should have been listed as a precursor twice

**Comment by bpbond** [comment]
@russellhz thanks! Yes please.
